openrtb_ext/request_wrapper_test.go

```diff
@@ -2253,6 +2254,21 @@ func TestRegExtUnmarshal(t *testing.T) {
 			expectGDPR:  ptrutil.ToPtr[int8](0),
 			expectError: true,
 		},
+		// GPC
```

Comment:
These tests look good as they adequately cover your changes to the `RegExt` `unmarshal` function. However, there are still other request wrapper changes that require test coverage:
1) Update `TestRebuildRegExt`, which will cover your `RegExt` `marshal` changes
2) Add a new test `TestRegExtGetGPCSetGPC` that is similar to `TestRegExtGetGDPRSetGDPR`